src/interfaces/common.ts

```diff
@@ -8,3 +8,16 @@ export enum BooleanEnum {
 	true = "true",
 	false = "false",
 }
+
+// Adding session information to express Request type
+declare global {
+	// eslint-disable-next-line @typescript-eslint/no-namespace
+	namespace Express {
+		interface Request {
+			session: {
+				jiraHost?: string;
+				githubToken?: string;
+			};
+		}
+	}
+}
```

Comment:
This used to be in src/frontend/app.ts, but since it's now being used by multiple files in the new route structure, I moved it into a more reusable area.